### PR TITLE
DropDownBox: update demo descriptions

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DropDownBox is one of multiple DevExtreme drop-down editors. To learn how to choose a DevExtreme drop-down editor, refer to the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```
Comment (dirk-pieterse) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]:
```suggestion
DropDownBox is one of multiple DevExtreme drop-down editors. For information on which DevExtreme drop-down editor you should use, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by natakazakova — DISMISSED** [review]

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+DropDownBox is an advanced drop-down editor that allows you to embed any content, including other components.
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]: This is probably OK. However, what do you think about the following version? An ability to include other component already implies that we have very few limitations regarding possible content...

```suggestion
DropDownBox is an advanced editor whose drop-down window can include other components.
```

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+DevExtreme offers multiple drop-down editor options. To find out which editor you should use, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]: Again, this is probably OK. However, this entry does not look consistent with the previous content. For example...

> DevExtreme offers multiple drop-down editor options.

Of what drop-down editor options do we speak here? Do we speak of options applicable to the `DropDownBox` we actually describe here? Do we speak of options common to all drop-down editors? Do we speak of other drop-down editors shipped with DevExtreme?

I haven't read the article you are referring to (How to Choose a Drop-Down Editor), however, its name implies that there we speak of other/all available drop-down editor types. If that's the case, the first sentence

> DevExtreme offers multiple drop-down editor options.

does not solve the task at hand.

Still, even if we find a proper solution for this conundrum, the non-consistency problem remains. At first, we speak of `DropDownBox`, and then all of the sudden start speaking of other editors available and then switch back to `DropDownBox` out of the blue.

If you that need to refer to the topic on other drop-down editors, you should've probably moved it after the next paragraph...

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+If the UI component embedded into the DropDownBox allows multiple selection, synchronize the DropDownBox value with the selected items. [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) are the same for every selection mode.
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]: First question I'm having at this point -- can I embed only one UI component into the `DropDownBox` component's drop-down window?

> [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) are the same for every selection mode.

Of what selection modes do we speak? It's not obvious from my standpoint.

This paragraph could have looked as follows:

```suggestion
If the UI component embedded into the DropDownBox component's drop-down window allows multiple selection, synchronize the component's value with the state of selected items as demonstrated in the following topic: [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/).
```

However, you can probably come up with something better than that.

File: apps/demos/Demos/DropDownBox/SingleSelection/description.md [hunk 5]
```diff
@@ -1,4 +1,6 @@
-The DropDownBox component consists of a text field and drop-down content. In this demo, the content is the TreeView and the DataGrid in single selection mode.
+DropDownBox is an advanced drop-down editor that allows you to embed any content, including other components. In this demo, the content is the TreeView and the DataGrid in single selection mode.
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/SingleSelection/description.md]: For example...

```suggestion
DropDownBox is an advanced editor with a drop-down window that can embed other components. In this demo, TreeView and the DataGrid components in single selection mode are embedded into the drop-down window.
```

File: apps/demos/Demos/DropDownBox/SingleSelection/description.md [hunk 6]
```diff
@@ -1,4 +1,6 @@
...
+
+DevExtreme offers multiple drop-down editor options. To find out which editor you should use, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/SingleSelection/description.md]:
```suggestion
DevExtreme ships with multiple drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+DropDownBox is an advanced editor whose drop-down window can include other components.
+
+DevExtreme ships with multiple drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]:
```suggestion
DevExtreme ships with multiple other drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
+
+If the UI component embedded into the DropDownBox component's drop-down window allows multiple selection, synchronize the component's value with the state of selected items as demonstrated in the following demo: [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/).
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]: > synchronize the component's value with the state of selected items as demonstrated in the following **demo**:

"Synchronization Instructions" is a strange name for a demo... It rather befits a documentation topic than a demo...

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
+<!--split-->
+
+In this demo, the DataGrid's **selection**.[mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#mode) and TreeView's [selectionMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#selectionMode) properties are used to enable multiple selection.
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]: Avoidable passive voice.

```suggestion
This demo uses the DataGrid's **selection**.[mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#mode) and TreeView's [selectionMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#selectionMode) properties to enable multiple selection.
```

File: apps/demos/Demos/DropDownBox/SingleSelection/description.md [hunk 10]
```diff
@@ -1,8 +1,12 @@
-The DropDownBox component consists of a text field and drop-down content. In this demo, the content is the TreeView and the DataGrid in single selection mode.
+DropDownBox is an advanced editor whose drop-down window can include other components.
+
+DevExtreme ships with multiple drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/SingleSelection/description.md]:
```suggestion
DevExtreme ships with multiple other drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```

File: apps/demos/Demos/DropDownBox/SingleSelection/description.md [hunk 11]
```diff
@@ -1,8 +1,12 @@
...
 
 To get started with the DevExtreme DropDownBox component, refer to the following step-by-step tutorial: [Getting Started with DropDownBox](/Documentation/Guide/UI_Components/DropDownBox/Getting_Started_with_DropDownBox/).
 <!--split-->
 
+In this demo, TreeView and the DataGrid components in single selection mode are embedded into the DropDownBox window.
```
Comment (dmitry-eliseev-devexpress) [inline apps/demos/Demos/DropDownBox/SingleSelection/description.md]: I believe we've got an adjective on our hands here...

```suggestion
In this demo, TreeView and the DataGrid components in single-selection mode are embedded into the DropDownBox window.
```

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]

File: apps/demos/Demos/DropDownBox/MultipleSelection/description.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+DropDownBox is an advanced editor whose drop-down window can include other components.
+
+DevExtreme ships with multiple other drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```
Comment (dirk-pieterse) [inline apps/demos/Demos/DropDownBox/MultipleSelection/description.md]:
```suggestion
DevExtreme ships with multiple drop-down editors. To find out which editor best suits your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by natakazakova — APPROVED** [review]

**Pull request merged**